ViewMoreRepliesCellViewModelTests.swift

```diff
@@ -0,0 +1,13 @@
+@testable import KsApi
+@testable import Library
+import Prelude
+import ReactiveExtensions_TestHelpers
+import XCTest
+
+internal final class ViewMoreRepliesCellViewModelTests: TestCase {
```

Comment:
Is this empty view model fine given it doesn't require any real functionality other than being tapped?

Reply:
Can remove this entire test file, we're not testing anything.

Reply:
Yeah, good point. Deleting.